Resources/ConfigPresets/SimpleStation14/Parkstation.toml

```
@@ -8,7 +8,7 @@ lobbyduration = 180
 role_timers = true
 fallbackpreset = "extended"
 map_pool = "ParkMapPool"
-panic_bunker.enabled = true
+panic_bunker.enabled = false
 admin_panic.enabled = true
```

```suggestion
admin_panic.enabled = false
```
?
As far as I know, this stops new people from joining when no Admins?

---

yes